src/store/getters.js

```diff
@@ -89,11 +89,7 @@ const loginButtonsArray = (state) => {
   return loginButtons
 }
 
-const supportedCurrencies = (state) => {
-  const returnArr = ['ETH', ...config.supportedCurrencies]
-  if (state.networkType.ticker !== 'ETH') returnArr.unshift(state.networkType.ticker)
-  return returnArr
-}
+const supportedCurrencies = () => config.supportedCurrencies
```

Comment:
we need to add custom ticker if it's not there.!!

Reply:
done